example/readme.md

```diff
@@ -0,0 +1,18 @@
+# C++ Examples
+
+A series of mini-apps to demonstrate specific features.
+Please refer to the respective sub-directories for more
+information.
+
+All examples can be built by setting up the standard
+CMake build process, e.g.
+
+    # go to arbor's source directory   
+    cd arbor
+    # set up a build dir
+    mkdir build
+    cd build
+    # configure (example options for an MPI-enabled build)
```

Comment:
Can we mention that it's also vectorization-enabled here? And below, instructions how to run, with a note saying to check the readme of each example?

Reply:
Certainly I can add SIMD here. But the individual READMEs are referenced above this example.

Reply:
That said, I made it a bit more clear ... hopefully.